force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -321,18 +392,18 @@ public with sharing class FlowControlLoopsCollections {
      * @return The salary of the employee, or null if the employee is not found in the Map.
      */
     public static Integer getSalary(String employeeName) {
-        // Initialize a Map
-
-        // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
-
-        // Get the salary of the employee
-
-        // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        Map<String, Integer> salariesByEmployee = new Map<String, Integer> {
+            'John Doe' => 50000,
+            'Jane Smith' => 60000,
+            'Sam Brown' => 55000,
+            'Alice Johnson' => 65000
+        };
+
+        if (salariesByEmployee.get(employeeName) == null) {
+            return null;
+        } else {
+            return salariesByEmployee.get(employeeName);
+        }
```

Comment:
Good diligence here but the default behaviour of a map is that if you call map.get('invalidKey'), the map will return null. So in this case, you could say `return salariesByEmployee.get(employeeName)` and if the employee is not in the map, the method will return `null`